engine/opts.go

```diff
@@ -28,11 +28,12 @@ const (
 )
 
 type ClientMetadata struct {
-	// ClientID is unique to every session created by every client
+	// ClientID is unique to each client. The main client's ID is the empty string,
```

Comment:
Note to self: fix this doc string; the client ID is now what used to be ModuleCallerDigest, so it's actually a non-unique content hash for all clients other than "main callers" from the host.